api/src/main/java/org/apache/gravitino/credential/CredentialConstants.java

```diff
@@ -0,0 +1,30 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ *  or more contributor license agreements.  See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.  The ASF licenses this file
+ *  to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.apache.gravitino.credential;
+
+/** Contains credential constants used by Gravitino server and client. */
+public class CredentialConstants {
+  /** Credential type in the credential. */
+  public static final String CREDENTIAL_TYPE = "credential-type";
+  /** Credential expire time at ms since the epoch. */
+  public static final String EXPIRE_TIME_AT_MS = "expire-time-at-ms";
```

Comment:
I think you can also define these two property keys in `Credential`, you don't have create a new class to maintain them.

Reply:
Also please change to "expire-time-in-ms".

Reply:
updated